superset/commands/security/update.py

```diff
@@ -39,17 +39,11 @@ def __init__(self, model_id: int, data: dict[str, Any]):
         self._roles = self._properties.get("roles", [])
         self._model: Optional[RowLevelSecurityFilter] = None
 
+    @transaction()
```

Comment:
I'm not sure why RLS doesn't have custom exceptions like most of the other commands.